src/structstore_shared.hpp

```diff
@@ -9,6 +9,12 @@
 
 namespace structstore {
 
+enum CleanupMode {
+    NEVER,
+    IF_LAST,
+    ON_CLOSE
```

Comment:
I feel like `ALWAYS` is more precise, as `IF_LAST` also means "cleanup on close". But I'm also fine with leaving it

Reply:
I considered using ```ALWAYS``` but somehow ended up using ```ON_CLOSE```. I'll happily change the flag name to ```ALWAYS```!